cnp.cpp

```diff
@@ -131,7 +131,7 @@ std::vector<std::string> find_elements_by_class(const std::string &html,
       if (closePos != std::string::npos) {
         std::string element =
             html.substr(pos, closePos + closingTag.length() - pos);
-        result.push_back(element);
+        result.push_back(get_element_text(element));
```

Comment:
Hey, why are you returning get_element_text here, We need to return proper vectors with <> intact. I don't think this is the right way.

Reply:
by elements dont we need to convert to just text? so just array of elements including the html tags (original code) is sufficient?